src/lib/Scenes/Show/Screens/Detail.tsx

```diff
@@ -100,10 +100,12 @@ export class Detail extends React.Component<Props, State> {
       },
     })
 
-    sections.push({
-      type: "shows",
-      data: show,
-    })
+    if (show.location) {
```

Comment:
should we use `show.has_location`? the logic behind `has_location` is slightly different and thats what defines if the show is online exclusive or not. What do you think? it's already exposed in MP for `Show`

Reply:
Ahhh, just saw this. Yeah that makes sense. I'll make a separate PR to use `has_location` instead